### ISSUE TITLE
extern() does not work with three.js >= r118

### ISSUE THREAD

**Issue by yandeu** [photonstorm/phaser#5343]
## Version
* Phaser Version: from 3.16.1 to 3.50.0-beta.5
* Operating system: Windows
* Browser: Firefox 81 and Chrome 85

## Description

The example [extern.js](https://labs.phaser.io/edit.html?src=src/bugs\extern.js) does not work with **three.js r118 and above**.
I have tried lot, but just could not make it work :/

## Example Test Code

You can test it by replacing the **line 24** of [extern.js](https://labs.phaser.io/edit.html?src=src/bugs\extern.js) with one of the lines below:

```js
// all three.js versions until r117 do work.
this.load.script('threejs', 'https://cdnjs.cloudflare.com/ajax/libs/three.js/r117/three.min.js');

// r118 and above, do not work
this.load.script('threejs', 'https://cdnjs.cloudflare.com/ajax/libs/three.js/r118/three.min.js');
```

**Comment by photonstorm** [comment]
So what did they change in r118 then? :)

**Comment by photonstorm** [comment]
Surely because of this:

WebGLRenderer automatically creates a WebGL 2 rendering context now (and fallbacks to WebGL 1 if necessary). If your project can only use WebGL 1, you can use WebGL1Renderer.

https://github.com/mrdoob/three.js/wiki/Migration-Guide#r117--r118

Although simply swapping for that renderer doesn't work on its own.

**Comment by yandeu** [comment]
I saw all the migration guides already and have tried a lot to make it work. But had no success.

Also, I use a WebGL2 context with Phaser by default.

There is the same issue with PixiJS https://github.com/mrdoob/three.js/issues/20384

In the meanwhile, I have switched to a new strategy to use Phaser and three.js together. I do now use 2 canvas. Even before three.js r118, Phaser and three.js did not work very well together (see https://github.com/enable3d/enable3d/issues/37). So switching to that strategy, solved more issues.

All [examples on enable3d.io](https://enable3d.io/examples.html) are now running with three.js r121.

The [First Phaser Game (3D Version)](https://enable3d.io/examples/first-phaser-game-3d-version.html) works as well as before :smiley:
